src/pf/parse.lua

```diff
@@ -838,7 +838,16 @@ local function parse_primitive_or_arithmetic(lexer)
    -- short for `not host vs and host ace` and which should not be
    -- confused with `not (host vs or ace)`."  For now we punt on this
    -- part of the grammar.
-   lexer.error('keyword elision not implemented %s', tok)
+   local msg =
+[[%s is not a recognized keyword. Likely causes:
+a) %s is a typo, invalid keyword, or similar error.
+b) An attempt to use keyword elision.
+Repeating the previous keyword is not implemented.
```

Comment:
Looks good to me.  Consider rewording to remove the phrase "keyword elision"; it's not something the user is going to understand.  i.e.  "Probably this is a typo that you should fix.  It could be, though, that you are using a shorthand form of pflang in which, for example, 'host a or b' means 'host a or host b'.  Pflua doesn't support this shorthand syntax yet; instead, write out the full form, without omitting keywords."

dunno, that was a pretty bad attempt and I'm sure that anything you come up with will be better :)